### PR TITLE
Remove average command

### PR DESCRIPTION
**Pull request by BenjaminTan99**
Removed the average command and converted it to class attribute.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -302,11 +302,10 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 
-1.  User requests to list applicants
-2.  MrTechRecruiter shows a list of applicants
-3.  User requests to delete a specific person in the list using the exact name
-4.  MrTechRecruiter deletes the person
-
+1.  User requests to list applicants.
+2.  MrTechRecruiter shows a list of applicants.
+3.  User requests to delete a specific person in the list using the exact name.
+4.  MrTechRecruiter deletes the person. <br>
```
Comment (Harry-Gao-H) [inline docs/DeveloperGuide.md:308]: I think double new lines can be used to start with a new line (consistent with other parts of the document)

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -302,10 +302,10 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 **MSS**
 
-1.  User requests to list applicants
-2.  MrTechRecruiter shows a list of applicants
-3.  User requests to delete a specific person in the list using the exact name
-4.  MrTechRecruiter deletes the person
+1.  User requests to list applicants.
+2.  MrTechRecruiter shows a list of applicants.
+3.  User requests to delete a specific person in the list using the exact name.
+4.  MrTechRecruiter deletes the person.
```
Comment (LMAOboxhack) [inline docs/DeveloperGuide.md:308]:
```suggestion
3.  User requests to delete a specific applicant in the list using the exact name.
4.  MrTechRecruiter deletes the applicant.
```

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -327,6 +327,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 1. User requests to see average rejection rates for all job positions.
```
Comment (LMAOboxhack) [inline docs/DeveloperGuide.md:328]:
```suggestion
1. User requests to see a list of all job positions
```

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -347,8 +348,9 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **MSS**
 
-1. User requests to see average rejection rates of a job position.
+1. User keys in the particular job position using the 'find' function.
```
Comment (LMAOboxhack) [inline docs/DeveloperGuide.md:351]:
```suggestion
1. User searches for a particular job position using the 'find' function.
```

File: docs/UserGuide.md [hunk 5]
```diff
@@ -80,7 +80,8 @@ Adds a person to the address book.
 Format: `add applicant n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS pos/POSITION​`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags (including 0)
+A person can have any number of tags (including 0). <br>
```
Comment (LMAOboxhack) [inline docs/UserGuide.md:83]:
```suggestion
An applicant can have any number of tags (including 0). <br>
```

**Review by LMAOboxhack — CHANGES_REQUESTED** [review]
Some minor changes to wording, otherwise LGTM!

**Review by LMAOboxhack — APPROVED** [review]
LGTM!

**Pull request merged**